TacTok/evaluate.py

```diff
@@ -63,6 +81,18 @@
     np.random.seed(opts.seed)
     random.seed(opts.seed)
 
+    # The identifier vocabulary
```

Comment:
So this is a copy of options.py that needs to be kept up-to-date whenever we change options.py? Is there any way to share any of that information across files? Curious because we will have our options changing fairly often

Reply:
Yeah there should be. The addition of this file isn't part of this pull request, since it already existed in CoqGym, I'm just keeping it up to date. The options don't perfectly match because there are some training-specific and evaluation-specific options, but it would be great to pull the shared options together into one file, and shouldn't be too hard.